oneview/resource_uplink_set.go

```diff
@@ -0,0 +1,404 @@
+// (C) Copyright 2016 Hewlett Packard Enterprise Development LP
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+package oneview
+
+import (
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceUplinkSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceUplinkSetCreate,
+		Read:   resourceUplinkSetRead,
+		Update: resourceUplinkSetUpdate,
+		Delete: resourceUplinkSetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"logical_interconnect_uri": {
```

Comment:
This is a required field. Please check the documentation once and mark the fields as required which are mentioned as mandatory fields. I see few mandatory fields mentioned as optional.

Reply:
Sorry missed it. Have added all the required tags.